collector.go

```diff
@@ -120,13 +122,23 @@ func (c collector) Collect(ch chan<- prometheus.Metric) {
 		if cached {
 			c = " (cached)"
 		}
-		log.Printf("Retrieved %s (%s), price: %f%s\n", qq.Symbol, qq.ShortName, qq.RegularMarketPrice, c)
+		log.Printf("Retrieved %s (%s), price: %f, volume: %s%s\n",
```

Comment:
Is there a specific reason why you convert the volume information and print it as a string (%s)? Why not print it directly as the others and avoid importing `strconv`?

Reply:
Oops, I'm sorry. I forgot `%d`. There is no other reason. I'll fix it.